src/ui/localization/locale/es.json

```
@@ -0,0 +1,57 @@
+{
+  "periods.week": "{{amount}} semana",
+  "periods.month": "{{amount}} mes",
+  "periods.year": "{{amount}} año",
+  "periods.day": "{{amount}} día",
+  "periods.weekShort": "{{amount}}wk",
+  "periods.monthShort": "{{amount}}mo",
+  "periods.yearShort": "{{amount}}yr",
+  "periods.dayShort": "{{amount}}d",
+  "periods.lifetime": "de por vida",
+  "periods.weekPlural": "{{amount}} semanas",
+  "periods.monthPlural": "{{amount}} meses",
+  "periods.yearPlural": "{{amount}} años",
+  "periods.dayPlural": "{{amount}} días",
+  "periods.weekFrequency": "cada semana",
+  "periods.monthFrequency": "cada mes",
+  "periods.yearFrequency": "cada año",
+  "periods.dayFrequency": "cada día",
+  "periods.unknownFrequency": "desconocida",
+  "periods.weekFrequencyPlural": "cada {{amount}} semanas",
+  "periods.monthFrequencyPlural": "cada {{amount}} meses",
+  "periods.yearFrequencyPlural": "cada {{amount}} años",
+  "periods.dayFrequencyPlural": "cada {{amount}} días",
+  "state_present_offer.product_title": "{{productTitle}}",
+  "state_present_offer.product_description": "{{productDescription}}",
+  "state_present_offer.product_price": "{{productPrice}}",
+  "state_present_offer.free_trial_duration": "Prueba gratuita por {{trialDuration}}",
+  "state_present_offer.price_after_free_trial": "{{productPrice}} después del final de la prueba",
+  "state_present_offer.renewal_frequency": "Se renueva {{frequency}}",
+  "state_present_offer.continues_until_cancelled": "Continúa hasta que se cancele",
+  "state_present_offer.cancel_anytime": "Cancela en cualquier momento",
+  "state_needs_auth_info.email_step_title": "Contraseña email de facturación",
```

I don't know where `Contraseña` came from, but it's definitely nonsense 😄 
```suggestion
  "state_needs_auth_info.email_step_title": "Email para facturación",
```

---

Actually, I’m wondering if just `Email` could be enough. By the context, it’s clear that it will be used to send receipts and information related to the purchase, isn’t it?

---

Maybe we can just use 
```suggestion
  "state_needs_auth_info.email_step_title": "Email de facturación",
```

wdyt @vicfergar ?